src/Aws/Laravel/AwsServiceProvider.php

```diff
@@ -28,7 +24,14 @@
  */
 class AwsServiceProvider extends ServiceProvider
 {
-    const VERSION = '1.1.0';
+    const VERSION = '1.5.0';
```

Comment:
I think we would need a different major version number. Since 2 and 3 are already taken, I'm not sure what the best course of action would be.